lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -258,6 +272,46 @@ def __init__(self, ufeSceneItem):
     def prependControlCreator(controlCreator):
         AETemplate._controlCreators.insert(0, controlCreator)
 
+    def orderSections(self, schemasAttributes):
+        '''
+        Choose the order in which the sections will be added to the AE template.
+        '''
```

Comment:
This is where sections are ordered. For now it is hard-coded. If we add a callback for order, this is where it would be. Probably filling the lists below, maybe adding a third list for middle items?